python/mxnet/module/sequential_module.py

```diff
@@ -149,7 +149,7 @@ def output_shapes(self):
         assert self.binded
         return self._modules[-1].output_shapes
 
-    def get_params(self):
+    def get_params(self, copy_to_cpu=True):
```

Comment:
Why is this argument needed here?

Reply:
Please check Line 167 in this file: arg, aux = module.get_params(). Do we need to pass in the copy_to_cpu value when calling get_params? arg, aux = module.get_params(copy_to_cpu)?